packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -176,60 +179,101 @@ Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs
 
 `nimble-table-column-text` will continue to present the same API it does today, but will derive from the base classes described above.
 
-#### Numeric column
+#### Number column
+
+Nimble will introduce `nimble-table-column-number-text` which formats a numeric field value and displays it as text. It will offer sufficient configuration to support use cases 1-4 above.
+
+##### API
 
-Nimble could introduce `nimble-table-column-numeric-text` which formats a numeric field value and displays it as text. It will offer sufficient configuration to support use cases 1-4 above.
+-   `prefix` - a string which will be appended before each value (e.g. `'$'`). Defaults to `''`.
+-   `suffix` - a string which will be appended after each value (e.g. `'%'` or `' V'`). Defaults to `''`. Spacing will be at the discretion of clients, but Nimble will recommend including a space before the unit except for symbol units like `%`, `"`, `°`, and `°C`. This [matches the Chicago Manual of Style](https://www.chicagomanualofstyle.org/book/ed17/part2/ch09/psec016.html) (requires VPN) which is NI's documentation style guide.
+-   `alignment` - a string value matching `"left"`, `"right"`, or `null` (the default, meaning `"automatic"`) which controls whether values and column headers are left or right aligned within the column. If set to `null` Nimble will choose left or right based on the value of `format`. Clients should select `right` if it is known that the decimal separators of all values in the column will align in the given the `format`.
```

Comment:
Good point. We should follow existing patterns so I updated the alignment and format values to use `undefined` instead of `null`. I had gotten confused because `element.getAttribute('non-existent-attribute')` returns `null` but I guess the property value accepted by the setter is `undefined`.